foreman_stub.py

```diff
@@ -72,4 +157,5 @@ def ping_heartbeat():
     return jsonify({"status": "ok", "response": "pong"})
 
 
-__name__ == '__main__' and app.run(host='0.0.0.0', port=int(os.getenv('PORT', 8080)), threaded=True, debug=DEBUG)
+if __name__ == '__main__':
```

Comment:
This is completely unrelated. And unneeded. Makes it hard to review.

Reply:
It's a separate commit. It was useful for me to have a pylint/pyflakes clean file so my editor makes it easier to spot bugs I'm introducing.